spec/features/users/create_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
```

Comment:
You need to organize your tests in a better way. Please create a file for each model or controller you are testing.

Reply:
Have Modified the test cases into more unit tests case with individual components.